app-core.js

```diff
@@ -55,20 +62,21 @@ function setupExpressServer() {
 async function ensureSystemSettingsInitiated() {
   logger.info("Checking Server Settings...");
 
-  await ServerSettingsDB.find({}).catch((e) => {
-    if (e.message.includes("command find requires authentication")) {
-      throw "Database authentication failed.";
```

Comment:
This code was essential to trigger database startup issues. I dont think removing this is smart